app/controllers/articles-controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectArticleByID } = require('../models/articles-model')
+
+exports.getArticleByID = (req, res, next)=>{
+    const articleID = req.params.article_id
+
+    selectArticleByID(articleID)
+    .then((rows)=>{
+        res.status(200).send({articles: rows})
```

Comment:
This should Ideally be on the key of article, rather than article(s), as it is only one article.